pephubclient/views/views.py

```diff
@@ -0,0 +1,34 @@
+class Views:
+    def __init__(self, jwt_data: str):
+        self._jwt_data = jwt_data
+
+    def get(self, namespace: str, name: str, tag: str, view_name: str):
+        ...
```

Comment:
you can use `pass` here to show it's not implemented
Or use ` raise NotImplementedError()` -- this shows clearly what you're intending